assets/js/modules/analytics/components/common/AccountCreate/GA4PropertyNotice.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Analytics GA4 Property Notice component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import SettingsNotice, { TYPE_INFO } from '../../../../../components/SettingsNotice';
+import Link from '../../../../../components/Link';
+
+export default function GA4PropertyNotice() {
+	return (
+		<SettingsNotice type={ TYPE_INFO }>
+			{ __( 'This will create both a Google Analytics 4 and Universal Analytics property.', 'google-site-kit' ) }
+			{ ' ' }
+			<Link
+				//  On ticket this says TODO
+				href="https://sitekit.withgoogle.com/documentation/ga4-analytics-property/"
```

Comment:
I'm assuming this is correct? (From the other PR)

Reply:
@danielgent Yes, the learn more link should always point to this resource - you can remove the `TODO` here.